packages/integration-karma/test/shadow-dom/event-in-shadow-tree/event-propagation.spec.js

```diff
@@ -199,8 +199,20 @@ describe('composed and bubbling event propagation in nested shadow tree', () =>
 
 describe('non-composed and bubbling event propagation in nested shadow tree', () => {
     let nodes;
+    let composedPath;
+    let expectedLogs;
     beforeEach(() => {
         nodes = createNestedShadowTree(document.body);
+        composedPath = [
+            nodes['span-manual'],
+            nodes['div-manual'],
+            nodes['x-shadow-tree'].shadowRoot,
+        ];
+        expectedLogs = [
+            [nodes['span-manual'], nodes['span-manual'], composedPath],
+            [nodes['div-manual'], nodes['span-manual'], composedPath],
+            [nodes['x-shadow-tree'].shadowRoot, nodes['span-manual'], composedPath],
+        ];
```

Comment:
We should probably move these back so they are closer to the test logic.

Reply:
sure